WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -59,89 +64,71 @@ private static EstimateSmartFeeResponse SimulateRegTestFeeEstimation(int confirm
 			return null;
 		}
 
+		/// <summary>
+		/// Estimates fees for 1w, 3d, 1d, 12h, 6h, 3h, 1h, 30m, 20m.
+		/// </summary>
 		public static async Task<AllFeeEstimate> EstimateAllFeeAsync(this IRPCClient rpc, EstimateSmartFeeMode estimateMode = EstimateSmartFeeMode.Conservative, bool simulateIfRegTest = false)
 		{
 			var rpcStatus = await rpc.GetRpcStatusAsync(CancellationToken.None).ConfigureAwait(false);
 			var mempoolInfo = await rpc.GetMempoolInfoAsync().ConfigureAwait(false);
 			var sanityFeeRate = mempoolInfo.GetSanityFeeRate();
-			var estimations = await rpc.EstimateHalfFeesAsync(new Dictionary<int, int>(), 2, 0, Constants.SevenDaysConfirmationTarget, 0, sanityFeeRate, estimateMode, simulateIfRegTest).ConfigureAwait(false);
-			var allFeeEstimate = new AllFeeEstimate(estimateMode, estimations, rpcStatus.Synchronized);
-			return allFeeEstimate;
-		}
 
-		public static async Task<RpcStatus> GetRpcStatusAsync(this IRPCClient rpc, CancellationToken cancel)
-		{
-			try
-			{
-				var bci = await rpc.GetBlockchainInfoAsync().ConfigureAwait(false);
-				cancel.ThrowIfCancellationRequested();
-				var pi = await rpc.GetPeersInfoAsync().ConfigureAwait(false);
+			var estmimations = (simulateIfRegTest && rpc.Network == Network.RegTest)
```

Comment:
```suggestion
			var estimations = (simulateIfRegTest && rpc.Network == Network.RegTest)
```

Reply:
Also, on line 74: `var sanityFeeRate = mempoolInfo.GetSanityFeeRate();`, the variable `sanityFeeRate` looks unused now.

Reply:
Also, not introduced by this PR but `*AllFee*` naming confuses me because I would either expect `*AllFees*` or `*FeeRates*` but "all fee" imho means "one fee" and not many.

Reply:
Ohh f***! good catch, thank you.

Reply:
Fixed.